### ISSUE TITLE
Blank profile modals in guilds, own contrib badge on everyone's profiles

### ISSUE THREAD

**Issue by Mihakuu** [HabitRPG/habitica#1479]
I wanted to check out people's profiles but... 
![hmmm](https://f.cloud.github.com/assets/3769301/1103715/bd1fb948-18bf-11e3-9e36-abddb4034461.PNG)
![lol](https://f.cloud.github.com/assets/3769301/1103716/bd21760c-18bf-11e3-801b-dd068d892f63.PNG)
![hmm](https://f.cloud.github.com/assets/3769301/1103717/bd213e4e-18bf-11e3-866c-505db8cd313a.PNG)
![look](https://f.cloud.github.com/assets/3769301/1103718/bd228786-18bf-11e3-943a-7894f0872830.PNG)

Basically everyone shows having the same achievements as myself. '
Refreshing helped with anons

**Comment by lefnire** [comment]
This is just in guilds, right? Party & tavern are still good. Yeah, I think
i need to add it in for guilds

**Comment by Mihakuu** [comment]
I noticed it in my guild but it still applies to tavern and party as well

**Comment by LordGrowl** [comment]
I can see everyone's stats and equipment all okay, but I'm having the same issue as Mihakuu in regards to the achievements: every profile I look at says "Elite Friend".  By the way, when you click on my profile I'm down as "LordGrowlTheAwesome"?  AWESOME!  =D

**Comment by Mihakuu** [comment]
The things we don't even know about ourselves LordGrowl. and LordGrowlTheAwesome is actually really cool. Mine just says the NPC I am.
Headless anons only happen when I have been on  habit for a bit and not refreshed. My guess is that it can't pull the character stuff because of time out, but on refresh I can see everyone just fine. They all just have my achievements. Though it was kind of funny because they had negative attributes which kind of makes sense that they'd have negative strength and defense without a head.

**Comment by paglias** [comment]
The chrome console shows any error? Open it with CTRL+Shift+J

**Comment by Mihakuu** [comment]
paglias apologies I really don't know what to look for, but I see nothing that says 'error' if that helps.

**Comment by lefnire** [comment]
yeah, looks like it shows everyone with your own contrib badge. I'm showing everyone as "Royal Blacksmith" - that part will be really easy to fix.

**Comment by lefnire** [comment]
pushing fix to the achievements issue

**Comment by lefnire** [comment]
pushing empty guild profiles fix
